tensorly/decomposition/_parafac2.py

```diff
@@ -17,8 +18,57 @@
 #          Yngve Mardal Moe
 
 
+def _update_imputed(tensor_slices, mask, decomposition=None, method="mode-2"):
```

Comment:
I'm not sure if we need these default values. In my experience, default values for arguments can make it very easy to "trip up" and forget to pass a value -- especially when we're refactoring. We can be explicit about `decomposition` being `None` when that's needed :)
```suggestion
def _update_imputed(tensor_slices, mask, decomposition, method):
```

Reply:
Thanks! This was addressed in a20b562526de762ff7e1f6296d92136e8348dd5b.